supabase/migrations/20240429145115_add_columns_birth_year_and_month_families_table.sql

```diff
@@ -0,0 +1,9 @@
+alter table "public"."families" add column "birth_month" smallint;
+
+alter table "public"."families" add column "birth_year" smallint;
+
+update "public"."families" set "birth_year" = 2024 - age;
```

Comment:
Is there a way to get current year in sql? I suppose we'll only make this transition this year but just to be safe